tests/patmat/virtpatmat_reach_sealed_unsealed.scala

```diff
@@ -18,4 +18,4 @@ object Test extends App {
   (true: Boolean) match { case true => case false =>  case _ => } // exhaustive, last case is unreachable
   (true: Boolean) match { case true => case false =>  case _: Boolean => } // exhaustive, last case is unreachable
   (true: Boolean) match { case true => case false =>  case _: Any => } // exhaustive, last case is unreachable
```

Comment:
Note - last pattern here is unreachable, but not _redundant_, so no warning is emitted for it in the current code. With intersection-based useless pattern checking, a warning is correctly emitted here.